documentation/docs/src/quick-start.md

```diff
@@ -34,26 +41,37 @@ namada ledger
 # can detach the tmux (Ctrl-B then D)
 ```
 
+For a more verbose output, one can run 
+```shell
+export ANOMA_TM_STDOUT='true'
+namada ledger
+```
+
 This should sync your node to the ledger and will take a while (depending on your computer). Subsequent commands (generating an account, etc.)  are unlikely to work until it is fully synced. Enquire the current block height with other participants to make sure you are synced in order to proceed.
 
 ## Account
 
-Generate a local key on disk
+Accounts on Namada are divided into two subcategories:
+* Implicit accounts (no attached Validity Predicate)
+* Established accounts (attached Validity Predicate)
 
-```shell
-# first, we make a keypair and the implicit account associated with it
-# namadaw address gen instead of key gen. Preferred because they both make a keypair but the former stores the implicit address for it too 
+In order to make transactions on Namada, an implicit account is required.
 
+An implicit account can be generated from the commands described below
+
+```shell
 namadaw address gen \
   --alias example-implicit
 
 ➜ Enter encryption password: 
 Successfully added a key and an address with alias: "example-implicit"
 
-# use it to make an established account
 ```
+An implicit account with alias "example-implicit" has now been successfully generated and stored locally in the wallet.
+
+It is possible to "upgrade" this implicit account to become an Established Account through submitting a transaction to do so. Note that this transaction, as with any transaction on Namada, will consume "gas", and will require a positive balance of NAM tokens.
```

Comment:
```suggestion
It is possible to "upgrade" this implicit account to become an Established Account through submitting a transaction to do so. Note that this transaction, as with any transaction on Namada, will consume "gas", and will require a positive balance of NAM tokens for the implicit account which is being used to create the established account.
```

Reply:
btw do you know if implicit accounts are charged gas now? i.e. when an implicit account makes a transaction, is NAM actually deducted from its balance?